### PR TITLE
Update to React migration docs

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR updates the React SDK from version 4 to version 5, focusing on migration steps, installation changes, and interacting with contracts using the new `thirdweb` SDK.

### Detailed summary
- Updated installation process for React SDK v5
- Added new components for interacting with contracts
- Introduced ethers.js Adapter for compatibility with v5
- Updated sidebar links for migration guide

> The following files were skipped due to too many changes: `apps/portal/src/app/react/v5/sidebar.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/portal/src/app/react/v5/migrate/ethers-adapter/page.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# WIP# Ethers.js in v5
```
Comment (joaquim-verges) [inline apps/portal/src/app/react/v5/migrate/ethers-adapter/page.mdx:1]: Wip?

File: apps/portal/src/app/react/v5/migrate/ethers-adapter/page.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Similar to `signer`, you can retrieve the ethers provider in thirdweb v5 using the same adapter:
+```ts
+import { ethers5Adapter } from "thirdweb/adapters/ethers5";
+import { createThirdwebClient } from "thirdweb";
+
+const client = createThirdwebClient({
+  clientId: "...",
+});
+
+const provider = ethers5Adapter.provider.toEthers({ client, chainId });
```
Comment (joaquim-verges) [inline apps/portal/src/app/react/v5/migrate/ethers-adapter/page.mdx:42]: Can you also add these to the tsdoc ?

**Review by joaquim-verges — APPROVED** [review]
Just remember to remove the WIP

File: apps/portal/src/app/react/v5/migrate/ethers-adapter/page.mdx (hunk 1 above)
Comment (kien-ngo) [reply apps/portal/src/app/react/v5/migrate/ethers-adapter/page.mdx:1]: oh damn, let me fix real quick

**Pull request merged**